### ISSUE TITLE
Connection error/warning to app.posthog.com when calling ploomber

### ISSUE THREAD

**Issue by JMankewitz** [ploomber/ploomber#744]
Hey! Every time I call ploomber I get the following error:
```
error uploading: HTTPSConnectionPool(host='app.posthog.com', port=443): Max retries exceeded with url: /batch/ (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fe8a9496e50>: Failed to establish a new connection: [Errno 61] Connection refused'))
```
Full stack here with `ploomber interact`

```
(wordsense_pipeline) jmank@LOYCA wordsense_pipeline % ploomber interact
Loading pipeline...
100%|██████████████████████████████████████████████████████████████████████| 14/14 [00:00<00:00, 7160.13it/s]
Python 3.8.13 | packaged by conda-forge | (default, Mar 25 2022, 06:06:49) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.33.0 -- An enhanced Interactive Python. Type '?' for help.

error uploading: HTTPSConnectionPool(host='app.posthog.com', port=443): Max retries exceeded with url: /batch/ (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdb60750d30>: Failed to establish a new connection: [Errno 61] Connection refused'))
```

Doesn't seem to be breaking anything, but kind of annoying. Thanks!

**Comment by edublancas** [comment]
Thanks for your feedback!

@idomic: my guess is that either posthog is too slow or some proxy issue. is there a way to timeout the call? or perhaps hide the error message?

**Comment by edublancas** [comment]
this is fixed in `0.19.1` - I reproduced a similar error but not exactly the one you got. so let me know if you still see the message after upgrading
